package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "banka",
+  "version": "1.0.0",
+  "description": "A banking application that helps users perform banking transactions.",
+  "main": "index.js",
+  "scripts": {
+    "build": "babel server/src -d server/dist",
+    "dev": "nodemon ./server/src/app.js --exec babel-node",
+    "start": "nodemon --exec babel-node src/app.js",
+    "test": "mocha --require @babel/register server/test/tests.test.js --exit",
+    "lint": "eslint \"**/*.js\" --fix"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/oleesir/banka.git"
+  },
+  "author": "Olisa Emodi",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/oleesir/banka/issues"
+  },
+  "homepage": "https://github.com/oleesir/banka#readme",
+  "devDependencies": {
+    "@babel/cli": "^7.4.4",
+    "@babel/core": "^7.4.4",
+    "@babel/node": "^7.2.2",
+    "@babel/preset-env": "^7.4.4",
+    "@babel/register": "^7.4.4",
+    "chai": "^4.2.0",
+    "eslint": "^5.3.0",
+    "eslint-config-airbnb-base": "^13.1.0",
+    "eslint-plugin-import": "^2.17.2",
```

Comment:
This was not used in the `eslint` config. What was your reason for adding the package and what do you want to use it for?

Reply:
they said we should use the airbnb style guide, so i installed this[ npx install-peerdeps --dev eslint-config-airbnb-base] to give me the latest dependencies

Reply:
Great!